Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp

```diff
@@ -316,7 +282,7 @@ FontPlatformData::FontPlatformData(float size, WebCore::FontOrientation&& orient
 
 FontPlatformData::IPCData FontPlatformData::toIPCData() const
 {
-    auto ctFont = font();
+    auto ctFont = FontPlatformData::ctFont();
```

Comment:
why not simply `ctFont()`?

Reply:
It wouldn't compile. I can also rename `ctFont` to `font` and it will compile.

Reply:
ah, right, you can't use this-> because it is a static member, don't worry about it.

Reply:
I mean, I don't mind. :) I think it would be a minor improvement in readability, it just would tweak more of the lines in the diff. :p